crates/server/src/authority/catalog.rs

```diff
@@ -8,7 +8,8 @@
 // TODO, I've implemented this as a separate entity from the cache, but I wonder if the cache
 //  should be the only "front-end" for lookups, where if that misses, then we go to the catalog
 //  then, if requested, do a recursive lookup... i.e. the catalog would only point to files.
-use std::{borrow::Borrow, collections::HashMap, io, sync::Arc};
+use std::sync::Arc;
+use std::{borrow::Borrow, collections::HashMap, io};
```

Comment:
Nit: should stick to the previous style here. Backstory: IIRC the idea is that we're moving to one statement per crate imports, but I'm mostly keeping it locally consistent and sticking with one statement per module in modules where that is the prevailing style. I also like to slowly move to StdExternalCrate but generally leaning to including all workspace-local imports with the crate block.